spec/components/document_list_spec.rb

```diff
@@ -74,6 +74,26 @@ def component_name
     assert_select "#{attribute} time[datetime='2017-07-19T15:01:48Z']"
   end
 
+  it "renders a document list item even when public_updated_at is nil" do
+    render_component(
+      items: [
+        {
+          link: {
+            text: "Some news stories are timeless",
+            path: "/timeless-news",
+          },
+          metadata: {
+            document_type: "News Story"
```

Comment:
Should this also have `public_updated_at: nil` in the hash?

Reply:
Ha, yes it should. Thanks for catching that!